never-ending/src/app/NotableItem/NotableItem.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import './NotableItem.css'
 
-export const NotableItem = ({name, type, lore, img, race, date, outcome, alignment, population}) => {
+export const NotableItem = ({ imagePosition, name, type, lore, img, race, date, outcome, alignment, population }) => {
```

Comment:
curious about the image position property, Im assuming this is to meet the wireframe of ging back and forth. this is kinda how i thought to do it too.

Reply:
@Adam-Meza Yes, this is what toggles the image being on the right or the left. It utilizes the index of the array to assign either the left or right class name.